src/chart/treemap/TreemapSeries.ts

```diff
@@ -265,6 +266,15 @@ class TreemapSeriesModel extends SeriesModel<TreemapSeriesOption> {
                 textStyle: {
                     color: '#fff'
                 }
+            },
+            emphasis: {
+                disabled: false,
+                itemStyle: {
+                    color: 'rgba(0,0,0,0.9)', //'#5793f3',
+                    textStyle: {
+                        color: '#fff'
+                    }
```

Comment:
The text style in _emphasis_ state looks the same as the style in _normal_ state, so I think this option is also unnecessary,